### DISCUSSION TITLE
Password reset request - Return consistent message for both existent and non-existent accounts.

### DISCUSSION THREAD

**Discussion by netcatgirl** [Traewelling/traewelling#2352] [Ideas / Feature Requests]
Currently, when requesting a new password, it lets the user know if the account exists or not. 
>  We can't find a user with that e-mail address

and
>  We have e-mailed your password reset link. 

maybe the message could be changed to something like 

> If there is a user account associated with this email address, we have e-mailed a password reset link to {emailAddress}.

I read about it on the [owasp.org auth cheat sheet](https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html#authentication-and-error-messages) and thought it might apply here.

**Comment by MrKrisKrisu** [comment]
Thanks for the hint. Will be fixed in the next release. :)

See #4438
